app/components/SearchHistory/index.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import SearchHistoryItem from 'components/SearchHistoryItem';
+import { SearchHistoryList, SearchHistoryLabel, SearchHistoryWrapper } from './styles';
+
+const SearchHistory = ({ searchHistory }) =>
+  searchHistory.length > 0 && (
+    <SearchHistoryWrapper>
+      <SearchHistoryLabel>Onlangs gezocht op:</SearchHistoryLabel>
+      <SearchHistoryList>
+        {searchHistory.map(item => (
+          <SearchHistoryItem key={item.url} url={item.url} text={item.text} />
```

Comment:
It might still happen that `item.url` isn't a unique value which leads to warnings and possibly unexpected behaviour.

Reply:
You're right. Fixed.